model_compression_toolkit/gptq/keras/quantizer/soft_rounding/symmetric_soft_quantizer.py

```diff
@@ -16,17 +16,53 @@
 import tensorflow as tf
 import numpy as np
 
+from model_compression_toolkit import RoundingType
+from model_compression_toolkit import quantizers_infrastructure as qi
 from model_compression_toolkit.core.common import max_power_of_two
+from model_compression_toolkit.core.common.target_platform import QuantizationMethod
 from model_compression_toolkit.gptq.common.gptq_constants import PTQ_THRESHOLD, SCALE_PTQ, N_EPOCHS, \
     MAX_ITERATIONS_DEFAULT, SOFT_ROUNDING_GAMMA, SOFT_ROUNDING_ZETA, SOFT_ROUNDING_BETA, GPTQ_ITER, AUXVAR
 from model_compression_toolkit.gptq.keras.quantizer import quant_utils as qutils
-from tensorflow_model_optimization.python.core.quantization.keras.quantize_wrapper import QuantizeWrapper
-from tensorflow.python.framework.tensor_shape import TensorShape
 from typing import Dict, Any, List
 from model_compression_toolkit.core.common.constants import THRESHOLD, MIN_THRESHOLD
 from model_compression_toolkit.core.common.logger import Logger
-from model_compression_toolkit.core.keras.quantizer.base_quantizer import BaseTrainableQuantizer
+from model_compression_toolkit.gptq.keras.quantizer.base_keras_gptq_quantizer import BaseKerasGPTQTrainableQuantizer
+from model_compression_toolkit.gptq.keras.quantizer.quant_utils import power_of_two_max, clip
+from model_compression_toolkit.gptq.pytorch.quantizer.quant_utils import calculate_delta
+from model_compression_toolkit.quantizers_infrastructure import TrainableQuantizerWeightsConfig
+from model_compression_toolkit.quantizers_infrastructure.common.base_inferable_quantizer import mark_quantizer
+
+
+def soft_rounding_symmetric_quantizer(input_tensor: tf.Tensor,
+                                      auxvar_tensor: tf.Variable,
+                                      threshold_tensor: tf.Tensor,
+                                      num_bits: int,
+                                      signed: bool,
+                                      power_of_two: bool) -> tf.Tensor:
+    """
+    Quantize a tensor symmetrically for GPTQ quantizers.
+
+    Args:
+        input_tensor: Tensor to quantize. values of this tensor are not changed during gptq.
+        auxvar_tensor: Tensor that manifests the bit shift the weight due to gptq.
```

Comment:
I'm not sure I understand... maybe a typo?

Reply:
Sorry, this is a "copy-pasted" description that I didn't write and I didn't verified it.
There is a type, the actual sentence should be:
"Tensor that manifests the bit shift **of** the **quantized** weights due to gptq **training**".
Is that clearer?